auth/src/main/java/org/aerogear/mobile/auth/metrics/AuthMetricsProvider.java

```diff
@@ -0,0 +1,21 @@
+package org.aerogear.mobile.auth.metrics;
+
+import android.content.Context;
+
+import org.aerogear.mobile.metrics.interfaces.MetricsProvider;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class AuthMetricsProvider extends MetricsProvider {
```

Comment:
same as my comment above. I think it is better to rename it `MetricsDataProvider` to make it clear that they are only to provide data for metrics, not the metrics service itself.